code/modules/detectivework/detective_work.dm

```diff
@@ -81,6 +81,8 @@
 	return TRUE //we bloodied the floor
 
 /mob/living/carbon/human/add_blood_DNA(list/blood_dna, list/datum/disease/diseases)
+	if(HAS_TRAIT(src, TRAIT_LIGHT_STEP)) //the character is agile enough to don't mess their clothing and hands just from one blood splatter at floor
+		return TRUE
```

Comment:
So this proc, `add_blood_DNA` is _only_ called by walking over a blood splatter, and no other times?

Or does this change give LIGHT_STEP trait holders the ability to not get bloody anymore?

Reply:
Bloodysoles.dm is the only file that has references to `add_blood_DNA`, yeah.
But I still fear something could be overlocked, so I moved the code highter